ci/pipeline.yaml

```diff
@@ -383,6 +383,9 @@ jobs:
               - |
                 microdnf install skopeo
                 DEV_BUILD_IMAGE=delivery.instana.io/int-docker-agent-local/instana-agent-operator/dev-build
+                ICR_REPOSITORY=icr.io/instana/instana-agent-operator
+                ARTIFACTORY_REPOSITORY="${ARTIFACTORY_CONTAINER_DOCKER_URL}/instana-agent-operator"
```

Comment:
The quotes are not required, it's more a safe-guard I try to internalize, that whenever I put a variable into a string, I put quotes around it as otherwise the statement is not fail-safe and might cause unintended behavior, as bash is not always great in putting out good error messages.